packages/rangelink-vscode-extension/src/__tests__/services/LinkGenerator.test.ts

```
@@ -496,5 +498,35 @@ describe('LinkGenerator', () => {
+        bindPerformed: false,
+      });
+      const strategies = captureSendStrategies<typeof link>(mockSendRouter.sendToDestination);
+
+      await generator.createLink();
+
+      expect(mockAdapter.getActiveTextEditorUri).toHaveBeenCalledWith();
+      expect(mockAdapter.getWorkspaceFolder).toHaveBeenCalledWith(mockDoc.uri);
+      expect(mockLogger.info).toHaveBeenCalledWith(
+        { fn: 'generateLinkFromSelection', formattedLink: link },
+        'Generated link: src/file.ts#L1',
+      );
```

ESLint

[error] Replace `⏎········{·fn:·'generateLinkFromSelection',·formattedLink:·link·},⏎········'Generated·link:·src/file.ts#L1',⏎······` with `{·fn:·'generateLinkFromSelection',·formattedLink:·link·},·'Generated·link:·src/file.ts#L1'`

(prettier/prettier)